### PR TITLE
Update BANNO.CDRENEW.V1.POW

### PR DESCRIPTION
**Pull request by tkainz**
Corrected missing comma in JSON output when cross account error - resulting in "oops" error instead of cross account error message.
Corrected terms display verbiage when term is only 1 (day or month) resulting on terms display of "1 months" instead of "1 month"
Corrected display of renewing term frequency - resulting in display of new renewing term with current term values instead of renewing term data from share.

### CODE REVIEW

File: certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW [hunk 1]
```diff
@@ -227,6 +231,7 @@ DEFINE
  SMINIMUMBALANCE           = MONEY
  SCURRENTSTATUS            = CHARACTER
  SCURRENTTERM              = CHARACTER
+ SRENEWTERM                = CHARACTER
```
Comment (rrobison44) [inline certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW:234]: I see that SCURRENTTERM gets set but is never referenced.  Should SCURRENTTERM be removed.

File: certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW [hunk 2]
```diff
@@ -1458,7 +1476,7 @@ PROCEDURE RETURNJSON1
   DO
    PRINT INDENT(1)+Q+"errorCode"+Q+": "+FORMAT(Q+"999"+Q,505)+","
    NEWLINE
-   PRINT INDENT(1)+Q+"loggingErrorMessage"+Q+": "+Q+"Cross account access attempt"+Q
+   PRINT INDENT(1)+Q+"loggingErrorMessage"+Q+": "+Q+"Cross account access attempt"+Q+","
    NEWLINE
```
Comment (rrobison44) [inline certificate-renewal/REPWRITERSPECS/BANNO.CDRENEW.V1.POW:1480]: I wonder how this tested out fine when I originally made this change.  Mystery.

**Review by rrobison44 — APPROVED** [review]
Looks good.

**Review by jucarson — APPROVED** [review]
Looks good to me. Thanks Tom!

**Review by jordan-keenan — APPROVED** [review]
The changes look good Tom!

**Pull request merged**